mousestyles/distribution/test/test_kolmogorov_test.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import print_function, absolute_import, division
+
+import numpy as np
+from scipy import stats
+from mousestyles.distribution.kolmogorov_test import perform_kstest
+from mousestyles.distribution.kolmogorov_test import get_travel_distances
+
+
+def test_get_travel_distances():
+    res = get_travel_distances(strain=0, mouse=0, day=0)
+    assert type(res) is np.ndarray
+    assert len(res) == 38845
```

Comment:
Does this length mean anything? If not I think it is better not to include it. You should instead make tests based on prior facts that you know to be true.